docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -169,7 +169,7 @@ The security mapping must provide one or more configuration settings:
   credential. If list cotains "*", then any key can be specified via extra credential.
```

Comment:
Please structure commits according to: https://github.com/trinodb/trino/blob/master/DEVELOPMENT.md#git-merge-strategy

Git history matters for us a lot, and this is also a matter of code review.

Reply:
Sure, after all your comments are resolved I will squash all my commits into a single one and commit that one